src/main/java/teammates/common/datatransfer/FeedbackResponseAttributes.java

```diff
@@ -162,7 +162,6 @@ public String getJsonString() {
     
     @Override
     public void sanitizeForSaving() {
-        this.feedbackSessionName = Sanitizer.sanitizeTitle(feedbackSessionName);
```

Comment:
@bamboo3250, I will take over this PR soon if you are busy. Let me know if you still want to work on this.